frb_codegen/src/ir/ident.rs

```diff
@@ -35,4 +35,10 @@ impl IrIdent {
     pub fn dart_style(&self) -> String {
         self.raw.to_case(Case::Camel)
     }
+
+    pub fn remove_raw_prefix(&mut self) {
+        if self.raw.starts_with("r#") {
+            self.raw = self.raw[2..].to_string();
+        }
```

Comment:
sth like

```suggestion
        self.raw = self.raw.strip_prefix("r#");
```

https://doc.rust-lang.org/std/string/struct.String.html#method.strip_prefix

(and tweak a bit about ownership)